TerminatorPlus-API/src/main/java/net/nuggetmc/tplus/api/agent/legacyagent/ai/IntelligenceAgent.java

```diff
@@ -100,22 +100,22 @@ private void task() throws InterruptedException {
     private void runGeneration() throws InterruptedException {
         generation++;
 
```

Comment:
I don't think this would work. Could you please use either ChatColor or convert it to components?
Please correct me if I'm wrong.